fix(ClickUp Node): Correct time estimate multiplier from 6000 to 60000

joshsorenson opened the pull request:
## Summary

<!--
Fixes a long-standing bug in the ClickUp node where the time estimate field was incorrectly multiplied by 6000 instead of 60000 when creating or updating a task. This caused users to enter inflated numbers (e.g., 300 instead of 30) to get the correct minute-based result.
-->

joshsorenson committed:
fix(ClickUp): Correct time estimate calculation from milliseconds to seconds in ClickUp node. Previously to get correct time you had to add a 0 to the end of the estimate due to a missing 0 in the calculation for the conversion.

Joffcom commented:
Hey @joshsorenson,

Thanks for the PR, We have created "GHC-1816" as the internal reference to get this reviewed.

One of us will be in touch if there are any changes needed, in most cases this is normally within a couple of weeks but it depends on the current workload of the team.

joshsorenson committed:
feat(Postgres Node): Add searchable schema and table selectors

Add search functionality to schema and table selectors in the Postgres node, similar to Airtable's implementation. This improves usability when working with databases that have many schemas or tables.

Changes:
- Add searchable property to schema and table resource locators
- Implement filter handling in schemaSearch and tableSearch methods
- Use case-insensitive search for better matching

joshsorenson committed:
just putting it back

michael-radency reviewed (changes requested):
Hello @joshsorenson , thanks for your PR

- please update PR name and add description reflecting changes in this PR - addition of search and reason why this fix needed
- on line 913 we also need to fix this issue
- logic for filtering very similar - could we extract it to separate helper?
- can you add tests for updated search functions?

michael-radency committed:
Merge branch 'master' of https://github.com/n8n-io/n8n into pr/joshsorenson/15031

ShireenMissi commented:
Hi @joshsorenson 
 Thanks for the contribution! Closing this PR as it’s been inactive for a while and appears to be stale.

If you’d like to pick this back up, feel free to reopen or open a new PR. We’re happy to revisit it.
